Tensile/TensileCreateLibrary.py

```diff
@@ -924,6 +922,179 @@ def writeSolutionCall(solutionName, problemType):
   s += ", stream, numInputEvents, inputEvents, outputEvent )"
   return s
 
+##############################################################################
```

Comment:
Why is all this code being added to TensileCreateLibrary?
Looks like this code is being copied from elsewhere - now we have 2 copies of this code to maintain?

Reply:
This code provides functionality to separate out the generation of solutions from the full Benchmark process. As part of a conversation we do not in practice use all the bench making steps; this code simplifies the overhead and complexity involved in the full process for use in the modernization by providing a means of solution generation without building a complex object to host the such generation.

Reply:
Yes. I still have not addressed this. My comment was an explanation of the purpose of exposing the Benchmarking Internals.

Reply:
I fixed this  in the latest commits